files/vite.config.mjs

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'vite';
+import { extensions, ember, classicEmberSupport } from '@embroider/vite';
+import { babel } from '@rollup/plugin-babel';
+
+export default defineConfig({
+  plugins: [
+    ember(),
+    babel({
+      babelHelpers: 'inline',
+      extensions,
+    }),
+  ],
+  build: {
+    rollupOptions: {
+      input: {
+        tests: 'tests/index.html',
```

Comment:
what happens if you visit `/` instead of `/tests`?

Reply:
The tentative plan is to be able to have an index.html there to let you interact directly with a test app. See ef4/tracked-params for an example.